.github/workflows/pr-preview.yaml

```
@@ -0,0 +1,527 @@
+              kubectl delete ingress $ingress_name --ignore-not-found=true
+              echo "   ✅ Cleaned up orphaned ingress"
+            fi
+          done
+
+          # Count only healthy PR deployments (excluding the current one if it exists)
+          HEALTHY_COUNT=0
+          kubectl get deployments -l app.kubernetes.io/component=pr-preview \
+            -o jsonpath='{range .items[*]}{.metadata.labels.pr-number}{" "}{.status.availableReplicas}{"\n"}{end}' | \
+          while read pr_num available_replicas; do
+            if [ -z "$pr_num" ]; then continue; fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)